Assets/Scripts/Combat/ActionRegenerate.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ActionRegenerate : Action
+{
+    protected const int PHASE_NONE = 0;
+    protected const int PHASE_ANIMATING = 1;
+
+    protected int phase = PHASE_NONE;
```

Comment:
I would change these to enumeration instead of int

Reply:
Good idea.